jobqueue-worker.pl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
```

Comment:
`Bugzilla::Install::Filesystem` needs to be updated to ensure this is set to `OWNER_EXECUTE`